account_credit_control/README.rst

```diff
@@ -1,11 +1,16 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
 Credit Control
 ==============
 
 Configuration
 -------------
 
 Configure the policies and policy levels in ``Accounting  > Configuration >
```

Comment:
Maybe you should put both terms. If you only install module _account_, you have "Invoicing". When you install _account_accountant_, then the menu is renamed to "Accounting".